client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx

```diff
@@ -67,7 +67,7 @@ const InvestigatorsTable: React.FC<Props> = ({ investigators, selectedRow, setSe
                                 {
                                     Object.keys(TableHeaders).map((cellHeader: string) => (
                                         <TableCell key={cellHeader}>
-                                            { getTableCell(investigator, cellHeader) }
+                                            { getTableCell(investigator, cellHeader.toLocaleLowerCase()) }
```

Comment:
what the reason for the casting ?

Reply:
the json object returned from DB has lowercase property name.  I was forced to do that.